### ISSUE TITLE
Number of training epochs for ImageNet

### ISSUE THREAD

**Issue by VishaalMK** [pathak22/context-encoder#23]
Hi, I would like to double check on the number of epochs required to train the context encoder for center region dropout on ImageNet. I have noticed that [train.lua](https://github.com/pathak22/context-encoder/blob/master/train.lua) uses a default value of 20, but it would be helpful to know the exact value used to produce the results in [demo.lua](https://github.com/pathak22/context-encoder/blob/master/demo.lua)  (using the pre-trained model) so that I could use them as a heuristic for my training. Thanks
(Just so that I could be sure that my results are equivalent to yours)
